### ISSUE TITLE
Regarding the loss function as specified in the "histo_keras" notebook

### ISSUE THREAD

**Issue by sayakpaul** [vaishleshik/histology_classifier#1]
Hi @cmacus. This issue is regarding the loss function you have used in the above-mentioned notebook. Referring to the following code:

```python
model = Sequential()
model.add(Conv2D(32, (3, 3), padding='same',
                 input_shape=(256,192,3)))
model.add(Activation('relu'))
model.add(Conv2D(32, (3, 3)))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))
model.add(Conv2D(64, (3, 3), padding='same'))
model.add(Activation('relu'))
model.add(Conv2D(64, (3, 3)))
model.add(Activation('relu'))
model.add(MaxPooling2D(pool_size=(2, 2)))
model.add(Dropout(0.25))
model.add(Flatten())
model.add(Dense(512))
model.add(Activation('relu'))
model.add(Dropout(0.5))
model.add(Dense(len(classes), activation='sigmoid'))
model.compile(optimizers.rmsprop(lr=0.0001, decay=1e-6),loss="binary_crossentropy",metrics=["accuracy"])
```

You are using **binary_crossentropy** as the loss function (without incorporating **OVA** classification paradigm). Why is that? It is vivid that the problem is a multi-class classification problem. Should not we be using **categorical_corssentropy** here (I used it in my notebook)?

**Comment by cmacus** [comment]
Hmm. Thanks, That's a residue from copy pasting from old code. But I think @magus96  has pointed it out (and probably corrected) in his fork.

**Comment by sayakpaul** [comment]
Okay. I have done what you conveyed in the email today. Will push the notebook after finalizing.
